packages/engine/paima-standalone/src/utils/input.ts

```diff
@@ -17,6 +17,7 @@ import {
 import { importOpenApiJson, importTsoaFunction } from './import.js';
 import type { Template } from './types.js';
 import RegisterRoutes, { EngineService } from '@paima/rest';
+import { BaseConfigWithoutDefaults } from '@paima/utils/src/config/loading.js';
```

Comment:
don't use this kind of import style (referencing a .js file from an NPM path). It can cause issues with compilation units